dashboard/src/routeTree.gen.ts

```diff
@@ -11,38 +11,62 @@
 // Import Routes
 
 import { Route as rootRoute } from './routes/__root'
+import { Route as TreeRouteImport } from './routes/tree/route'
 import { Route as IndexImport } from './routes/index'
 import { Route as TreeIndexImport } from './routes/tree/index'
+import { Route as TreeTreeIdRouteImport } from './routes/tree/$treeId/route'
 import { Route as TreeTreeIdIndexImport } from './routes/tree/$treeId/index'
+import { Route as TreeTreeIdTestRouteImport } from './routes/tree/$treeId/test/route'
+import { Route as TreeTreeIdTestTestIdRouteImport } from './routes/tree/$treeId/test/$testId/route'
 import { Route as TreeTreeIdTestTestIdIndexImport } from './routes/tree/$treeId/test/$testId/index'
 import { Route as TreeTreeIdBuildBuildIdIndexImport } from './routes/tree/$treeId/build/$buildId/index'
 
 // Create/Update Routes
 
+const TreeRouteRoute = TreeRouteImport.update({
+  path: '/tree',
+  getParentRoute: () => rootRoute,
+} as any)
+
 const IndexRoute = IndexImport.update({
   path: '/',
   getParentRoute: () => rootRoute,
 } as any)
 
 const TreeIndexRoute = TreeIndexImport.update({
-  path: '/tree/',
-  getParentRoute: () => rootRoute,
+  path: '/',
+  getParentRoute: () => TreeRouteRoute,
+} as any)
+
+const TreeTreeIdRouteRoute = TreeTreeIdRouteImport.update({
+  path: '/$treeId',
+  getParentRoute: () => TreeRouteRoute,
 } as any)
 
 const TreeTreeIdIndexRoute = TreeTreeIdIndexImport.update({
-  path: '/tree/$treeId/',
-  getParentRoute: () => rootRoute,
+  path: '/',
+  getParentRoute: () => TreeTreeIdRouteRoute,
+} as any)
+
+const TreeTreeIdTestRouteRoute = TreeTreeIdTestRouteImport.update({
+  path: '/test',
+  getParentRoute: () => TreeTreeIdRouteRoute,
+} as any)
+
+const TreeTreeIdTestTestIdRouteRoute = TreeTreeIdTestTestIdRouteImport.update({
```

Comment:
nit: is there any way we can improve those variables names to avoid this much repetition?

Reply:
This is a generated file, so we can't, but we also don't need to, as we won't be manipulating this code.